chanzuckerberg/single-cell-data-portal
# feat: pull latest ontology mappings from s3 on webhook trigger

## Reason for Change

- #4796

## Changes

- add github action that, on webhook trigger (to be created in single-cell-curation upon new ontology reference file release), pulls latest ontology mappings for backend + frontend in s3 and commits to repo + pushes to main
- Changed frontend constants to pull data from json fixtures rather than hardcoded into constants.ts file (but retained constant name)

## Testing steps

- Tested in branch (see [AUTO: pull latest ontology files after schema migration](https://github.com/chanzuckerberg/single-cell-data-portal/commit/1e2361375ce9375d3fe287255406e5e7423d9e89) and the [action](https://github.com/chanzuckerberg/single-cell-data-portal/actions/runs/5521559298) that created it)

## Notes for Reviewer
